Ag pub2priv

agoude opened the pull request:
- New guide(s) 

## Description

This guide used to be part of our docs, but was deleted then. We updated the content and made it available again.

## Mandatory information

- This Pull Request can be merged as soon as possible.
- Tech review being needed


- This Pull Request content should be replicated for the US OVHcloud documentation : YES<!-- https://support.us.ovhcloud.com/hc/en-us -->

agoude committed:
Create guide.en-gb.md

Changelog:
- creation of the file

agoude committed:
Create test

will be removed then

agoude committed:
Add image

Add object_storage_buckets_vrack_private.png

agoude committed:
Delete pages/storage_and_backup/object_storage/s3_interconnect_with_vRack/images/test

Delete test file

agoude committed:
Update guide.en-gb.md

agoude committed:
Create meta.yaml

new meta.yaml file

agoude committed:
Update index.md

update index file w/ s3_interconnect_with_vRack

agoude committed:
Update guide.en-gb.md

agoude committed:
Create guide.fr-fr.md

FR guide version

agoude committed:
Update guide.en-gb.md

add conclusion

agoude committed:
Update guide.fr-fr.md

typo

agoude committed:
Update guide.fr-fr.md

typos

agoude committed:
Update guide.en-gb.md

change arbo

agoude committed:
Update guide.fr-fr.md

change arbo

Y0Coss reviewed (changes requested):
@agoude , as explained in private, you should re-use the same folder, id and full_slug as the version which was deleted in #8086

agoude committed:
Update meta.yaml

reuse ID and slug from previous PR

agoude committed:
Create test

test file for new folder - will be removed

agoude committed:
Delete pages/storage_and_backup/connect_other_ressources_in_vrack_private_network directory

agoude committed:
Create meta.yaml

meta.yaml file in the corresponding folder

agoude committed:
Rename guide.fr-fr.md to guide.fr-fr.md

Move file to the right folder /connect_other_ressources_in_vrack_private_network/

agoude committed:
Rename guide.en-gb.md to guide.en-gb.md

Move file in the correct folder /connect_other_ressources_in_vrack_private_network/

agoude committed:
Rename object_storage_buckets_vrack_private.png to object_storage_buckets_vrack_private.png

Move image in the correct folder /connect_other_ressources_in_vrack_private_network/

agoude committed:
Delete pages/storage_and_backup/object_storage/s3_interconnect_with_vRack/meta.yaml

Remove older meta.yaml file

agoude committed:
Update index.md

Update index with correct path: storage_and_backup/object_storage/connect_other_ressources_in_vrack_private_network/

agoude commented:
> @agoude , as explained in private, you should re-use the same folder, id and full_slug as the version which was deleted in #8086

@Y0Coss just did all the modifications to match with your requirements. Should be hopefully fine now. Thanks!

Y0Coss committed:
Proofreading

lason-ovh commented on pages/storage_and_backup/object_storage/connect_other_ressources_in_vrack_private_network/guide.en-gb.md:
````diff
@@ -0,0 +1,79 @@
+} 
+```
+
+To set this new policy to your S3 user, please follow the different steps shared in the [Object Storage - Identity and access management](/pages/storage_and_backup/object_storage/s3_identity_and_access_management) guide, and finalise the interconnection between your Object Storage resources and those within your vRack private network.
````
maybe add some additional infos :
- there are exceptions regarding 'the bucket owner'  --> it could be a best practice to not use the bucket owner and create other users instead
- a policy has to be applied to each user accessing the buckets/objects

agoude replied:
> maybe add some additional infos :
> 
>     * there are exceptions regarding 'the bucket owner'  --> it could be a best practice to not use the bucket owner and create other users instead
> 
>     * a policy has to be applied to each user accessing the buckets/objects

Modifications done! thx @lason-ovh

agoude replied:
can you please approve the change? @lason-ovh

lason-ovh reviewed (commented):
see my comment

agoude committed:
Update guide.en-gb.md

Changelog:
- new details regarding bucket owner & policy association

agoude committed:
Update guide.fr-fr.md

Changelog:
- new details regarding bucket owner & policy association

lason-ovh reviewed (approved):
ok for me

Y0Coss committed:
Proofreading last additions

Y0Coss commented:
Thank you for your contribution @agoude
